utils/config_build_and_install.windows.sh

```diff
@@ -25,7 +25,8 @@ build_system_dependency()
     cmake ../ -DCMAKE_BUILD_TYPE=$BUILD_TYPE         \
               -DCMAKE_INSTALL_PREFIX=$INSTALL_PREFIX \
               -DBUILD_TESTING:BOOL=OFF               \
-              -DGEOS_ENABLE_TESTS=False
+              -DGEOS_ENABLE_TESTS=False \
+              ${@:3}
```

Comment:
what's this?

Reply:
Passing all trailing arguments as extra compilation flags. This allows per-project configuration where that makes more sense. For example, for google benchemark, `-DBENCHMARK_ENABLE_GTEST_TESTS:BOOL=OFF` gets passed as an extra argument to cmake:

```
  build_system_dependency google/benchmark master "$SOURCE_DIR/benchmark" -DBENCHMARK_ENABLE_GTEST_TESTS:BOOL=OFF
```

Just pushed the same change for the geos-specific flags.